systems/robotInterfaces/Side.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SYSTEMS_CONTROLLERS_SIDE_H_
+#define SYSTEMS_CONTROLLERS_SIDE_H_
+
+#include <array>
+#include <string>
+
+// adapted from https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Type_Safe_Enum
+class Side
```

Comment:
LeftRight sounds fine to me.  Seems like a lot of infrastructure for something so simple.  :)
On May 8, 2015, at 1:50 PM, Twan Koolen <notifications@github.com> wrote:

> In systems/robotInterfaces/Side.h:
> 
> > @@ -0,0 +1,33 @@
> > +#ifndef SYSTEMS_CONTROLLERS_SIDE_H_
> > +#define SYSTEMS_CONTROLLERS_SIDE_H_
> > +
> > +#include <array>
> > +#include <string>
> > +
> > +// adapted from https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Type_Safe_Enum
> > +class Side
> Any suggestions? We used RobotSide at IHMC, but I always found there was nothing robot-specific about left or right. Maybe just LeftRight?
> 
> —
> Reply to this email directly or view it on GitHub.
>